lang/LangSource/PyrObject.cpp

```diff
@@ -110,7 +110,7 @@ PyrSymbol *s_identitydictionary;
 PyrSymbol *s_linkedlist;
 PyrSymbol *s_sortedlist;
 PyrSymbol *s_array;
-PyrSymbol *s_list, *s_method, *s_fundef, *s_frame, *s_class;
+PyrSymbol *s_list, *s_rest, *s_method, *s_fundef, *s_frame, *s_class;
```

Comment:
> I suppose you'll have to getsym("Rest") on every invocation, which I suppose may also have a slight performance impact.

Very very minimal impact; this happens all over the codebase. I think this is a clear instance of premature optimization; the time you're saving is minimal, and compared to the technical debt of inserting new global members into distant files, not worth it.

> but why is IdentityDictionary intrinsic while Rest should not be?

An intrinsic class is one that's defined at the C++ level; as far as I can tell there is no other difference. Some techno-archaeology shows that some classes like `Rect` were at one point slated to become intrinsic. The reason more classes don't become intrinsic is because nobody has taken the time to write that code. :)